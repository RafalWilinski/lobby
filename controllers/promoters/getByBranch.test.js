const getByBranch = require("./getByBranch");

describe("GET /promoters", () => {
  it("should return 200 and array of promoters", async () => {
    const ctx = {
      request: {}
    };

    await getByBranch(ctx);

    expect(ctx.body.promoters.length).toBeGreaterThan(0);
  });
});
