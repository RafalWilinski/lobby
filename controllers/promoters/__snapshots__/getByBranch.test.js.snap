// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /promoters should return 200 and array of promoters 1`] = `
Array [
  Object {
    "createdAt": null,
    "degree": "prof",
    "firstName": "Jerzy",
    "id": 1,
    "lastName": "Nawrocki",
    "updatedAt": null,
  },
]
`;
